Automattic/wp-calypso
# Nav Unification: add Classic Dark color scheme (default ready); prepare Classic Blue for default theme change;

#### Changes proposed in this Pull Request

This PR
* introduces Classic Dark as a new color scheme option (similar to the current default color scheme it has all properties set already so we can easily swap it in as the new default when we launch nav-unification)
* prepares Classic Bright for the eventual default color scheme switch to Classic Dark by adding missing colors previously inherited from Classic Bright

Context: 

We will allow customers to get familiar with the new colours ahead of the nav unification launch by introducing a new non-default Classic Dark color scheme using the nav unification colors (Decision: stakeholder decision in Ajax backlog grooming meeting on Thu Nov 5th.).

Related issue: https://github.com/Automattic/wp-calypso/issues/47266


#### Testing instructions



* Check out branch locally, `yarn && yarn start`, visit http://calypso.localhost:3000/me/account
* OR use the [calypso.live link](https://calypso.live/?branch=feature/new-default-color-scheme) (easier!)
* Select the Classic Dark color scheme in /me/account
* Compare against `flags=nav-unification` colors
* Navigate around Calypso and see if everything looks as expected
* Change color scheme to Classic Blue and see if it still match what you see in production.


Fixes https://github.com/Automattic/wp-calypso/issues/47266
